pilot/cmd/pilot-agent/main.go

```diff
@@ -341,6 +341,20 @@ func waitForCompletion(ctx context.Context, fn func(context.Context)) {
 	wg.Done()
 }
 
+// TODO: move this to API in 1.2
+func handleTDDEnvVar() time.Duration {
+	tddEnvVar, found := os.LookupEnv("TERMINATION_DRAIN_DURATION_SECONDS")
```

Comment:
could we move the constant to a more common place? Would be good to get all the variables in one place eventually.

Reply:
We actually do have a place - pkg/features/pilot :-)